aiidalab_launch/__main__.py

```diff
@@ -67,6 +68,13 @@
 pass_app_state = click.make_pass_decorator(ApplicationState, ensure=True)
 
 
+def exception_handler(exception_type, exception, traceback):  # noqa: U100
```

Comment:
Could we avoid the `noqa` with this?
```suggestion
def exception_handler(exception_type, exception, _):  # noqa: U100
```

Reply:
Cool, didn't know about this. I have also removed one other such instance just below.

Reply:
Uh, I take that back. The suggestion just produces a different lint error
```
U101 Unused argument '_'
```
So I reverted the change. This also explains why the exact same noqa is used just couple lines below.

Reply:
Ok, well, we tried. 😉